src/sentry-global-search/lib/types.ts

```diff
@@ -0,0 +1,123 @@
+import { ObjectWithObjectID } from '@algolia/client-search';
+
+export type Site = 'docs' | 'develop' | 'help-center' | 'blog';
+
+export type SearchHit = ObjectWithObjectID & {
+  /**
+   * URL of the result
+   */
+  url: string;
+  /**
+   * Title of the result
+   */
+  title: string;
+  /**
+   * ID of the result if available
+   */
+  id?: string;
+  /**
+   * Anchor ID of the result if available
+   */
+  anchor?: string;
+  /**
+   * String or path to the "result section"
+   */
+  section?: string | { full_path: string };
```

Comment:
This part is a little questionable. I think this is probably part of our search configuration.

Reply:
This is because we use `section` as a string in all of the gatsby indexes we control, but this happens to coincide with `section` in the zendesk index which is scraped and beyond our (at least easy) control.